base/api-deployment.yaml

```diff
@@ -164,6 +160,10 @@ spec:
               value: '$(SENDGRID_API_KEY)'
             - name: SQLALCHEMY_DATABASE_URI
               value: '$(POSTGRES_SQL)'
+            - name: SQLALCHEMY_DATABASE_READER_URI
+              value: '$(SQLALCHEMY_DATABASE_READER_URI)'
+            - name: SQLALCHEMY_POOL_SIZE
+              value: '$(SQLALCHEMY_POOL_SIZE)'
```

Comment:
Ah! I should have expanded the whole file to see in which section the new variables were added in. Github did not show these contextual sections.

Reply:
My bad as well. It happens!